tests/test_cluster.py

```diff
@@ -1989,6 +2025,29 @@ def test_persistent_security_group(self):
                                                     }
                                                 }})
 
+    def test_bastion_node_public_dns(self):
```

Comment:
could we also include a test for the `private-subnet-security-group` updated along with the `bastion_node_public_dns` on a cluster within the VPC with private subnet?